quinn-proto/src/connection/mod.rs

```diff
@@ -1230,7 +1230,9 @@ impl Connection {
         self.spaces[self.highest_space].ping_pending = true;
     }
 
-    #[doc(hidden)]
+    /// Update traffic keys
+    ///
+    /// The reason this is public is primarily for testing purposes.
     pub fn initiate_key_update(&mut self) {
```

Comment:
Should we unify the naming for consistency? I lean towards `force_`.

Reply:
Yes, agreed that would be nicer.

Reply:
Done.